config-default.yml

```
@@ -231,11 +232,12 @@ guild:
         - *HELPERS_ROLE
 
     webhooks:
-        talent_pool:    569145364800602132
-        big_brother:    569133704568373283
-        reddit:         635408384794951680
-        duck_pond:      637821475327311927
-        dev_log:        680501655111729222
+        talent_pool:                  569145364800602132
+        big_brother:                  569133704568373283
+        reddit:                       635408384794951680
+        duck_pond:                    637821475327311927
+        dev_log:                      680501655111729222
+        python_news: &PYNEWS_WEBHOOK  701731296342179850
```

Webhook for production:
```suggestion
        python_news: &PYNEWS_WEBHOOK 704381182279942324
```